app/controllers/ctc/offboarding/actc_without_dependents_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Ctc
+  module Offboarding
+    class ActcWithoutDependentsController < CtcController
+      helper_method :illustration_path, :illustration_folder, :prev_path
+
+      layout "intake"
+
+      def show; end
+
+      def file_with_gyr
+        current_intake.touch(:clicked_file_with_gyr_at)
+        redirect_to url_for(host: MultiTenantService.new(:gyr).host, controller: "/public_pages", action: 'home', params: { source: 'received_ctc' })
```

Comment:
this will create a url that looks like 
getyourrrefund.org/en/?source=received_ctc

if you use the recently added root_with_source_url(host: ..., source: "received_ctc"),
it will create the url as getyourrrefund.org/en/received_ctc, which stores the source param then issues a redirect to home so that the source param isn't maintained in the url.